Cargo.toml

```diff
@@ -63,7 +63,7 @@ hex = "^0.4"
 ignition-config = ">= 0.3, < 0.6"
 lazy_static = "^1.4"
 libc = "^0.2"
-nix = { version = ">= 0.24, < 0.28", "default_features" = false, "features" = [ "dir", "ioctl", "mount", "process", "sched", "signal", "user"] }
+nix = { version = ">= 0.24, < 0.30", "default_features" = false, "features" = [ "dir", "ioctl", "mount", "process", "sched", "signal", "user"] }
```

Comment:
I think that we should change this range, the new bottom should be 0.29.0 right?